src/plugins/vis_type_timeseries/server/lib/timeseries_visualization_client_wrapper.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  SavedObjectsClientContract,
+  SavedObjectsClientWrapperFactory,
+  SavedObjectsClientWrapperOptions,
+  SavedObjectsCreateOptions,
+  SavedObjectsErrorHelpers,
+} from '../../../../core/server';
+import { getDataSourceEnabled } from './services';
+
+export const TIMESERIES_VISUALIZATION_CLIENT_WRAPPER_ID = 'timeseries-visualization-client-wrapper';
+export const TIMESERIES_VISUALIZATION_CLIENT_WRAPPER_PRIORITY = 11;
```

Comment:
Can you add comment to describe why you are using 11 priority?

Reply:
Did you confirm that we are not using this priority in any other plugin?

Reply:
Added comment regarding using priority. I checked how other plugins were using priority and they used `-4,-3,-2,-1,-,1`